sky/provision/kubernetes/utils.py

```diff
@@ -370,21 +370,51 @@ class KarpenterLabelFormatter(SkyPilotLabelFormatter):
     """
     LABEL_KEY = 'karpenter.k8s.aws/instance-gpu-name'
 
+class NebiusLabelFormatter(GPULabelFormatter):
+    """Nebius label formatter
+    Nebius uses the label `node.kubernetes.io/instance-type` as the key, 
+    and a lowercase accelerator str as the value.
+
+    ex. "node.kubernetes.io/instance-type=gpu-h100-sxm"
+    Nebius docs: https://docs.nebius.com/compute/virtual-machines/types/
+    """
+    LABEL_KEY = 'node.kubernetes.io/instance-type'
+
+    @classmethod
+    def get_label_key(cls, accelerator: Optional[str] = None) -> str:
+        return cls.LABEL_KEY
+
+    @classmethod
+    def get_label_keys(cls) -> List[str]:
+        return [cls.LABEL_KEY]
+
+    @classmethod
+    def get_label_value(cls, accelerator: str) -> str:
```

Comment:
can we raise a `ValueError` here if an accelerator is not specified to be an H100. Also might make sense to define in the class the set of supported accelerators (it'd just be `[H100]` in this case)